compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -1208,6 +1372,32 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
            [](::concretelang::clientlib::KeySet &keySet) {
              return pybind11::bytes(keySetSerialize(keySet));
            })
+      .def("serialize_lwe_secret_key_as_glwe",
+           [](::concretelang::clientlib::KeySet &keySet, size_t keyIndex,
+              size_t glwe_dimension, size_t polynomial_size) {
```

Comment:
why we need glwe_dimension and polynomial_size here, could we infer directly from the secretKey itself?

Reply:
You can interpret it with different set of parameters (e.g glwe_dim=1, poly_size=2048 should also work with glwe_dim=2, poly_size=1024). I agree that we will be mostly working with glwe_dim=1, so poly_size would be equal to the lwe_size which is present in the key. However, this makes the func more generic (can view it as a glwe of different parameters), and these information are passed all the way from TFHErs parameters, which makes sure the key is what TFHErs would expect. We can also have asserts for the glwe_dim if that helps.